### ISSUE TITLE
Is it possible for `exif.Orientation` to be `undefined`?

### ISSUE THREAD

**Issue by OliverJAsh** [mattiasw/ExifReader#140]
### Description

According to the TypeScript types, `exif.Orientation` is a required property:

```ts
import { load } from 'exifreader';

const result = load(data, { expanded: true });

if (result.exif !== undefined) {
    result.exif.Orientation.value; // number
}
```

We are using this library in production at [Unsplash](https://unsplash.com/) (thank you!) and we have noticed that a few users are running into an exception because `Orientation` is actually `undefined`, which is not expected because the type definitions say it will always exist (as per my example above).

Unfortunately I have not been able to reproduce this myself. Maybe it's something to do with the photo that is being uploaded.

If there is a good reason why `Orientation` would be `undefined`, perhaps we should update the type definitions. What do you think?

For reference, here is a previous issue that is similar (but not the same). This one is now fixed: https://github.com/mattiasw/ExifReader/issues/135.

**Comment by mattiasw** [comment]
Hi! I would say it's not weird at all that an image does not have an Orientation value. The library only adds the tags that the file contains and there definitely are situations where that value can be missing. This is true for a lot of tags though. If the definitions say it's required I agree they need to be updated. I don't use Typescript myself (and didn't make the type definitions file) so I'm not sure what needs to be changed though. I could probably make an educated guess but I think a safer way would be for someone to make a PR, or at least guide me. Do you think you could help with that?

And Unsplash is an awesome service, so thank you too! :-)

**Comment by OliverJAsh** [comment]
Thanks for the reply! ❤️ 

I made a PR: https://github.com/mattiasw/ExifReader/pull/141

**Comment by mattiasw** [comment]
Thanks! I merged it and it is now published to npm as version 3.14.0.
